server/models/List.js

```diff
@@ -0,0 +1,20 @@
+const mongoose = require('mongoose')
+
+const ListSchema = new mongoose.Schema({
+  name: {
+    type: String,
+    required: true,
+  },
+  image: {
+    type: String,
+  },
+  products: [{ type: mongoose.Schema.Types.ObjectId, ref: 'Product' }],
+  userId: {
+    type: String,
```

Comment:
I think since this is a reference to a userId the type should be ```mongoose.Schema.Types.ObjectId``` and also should have ```ref:'User'``` as an attribute